src/styles/shared/_buttons.scss

```diff
@@ -31,9 +31,6 @@
   cursor: pointer;
   user-select: none;
   text-decoration: none;
-  transition-property: background, border, box-shadow;
-  transition-duration: var(--p-override-none, duration());
```

Comment:
What's the timeline for removing these? Won't that break apps using the component if they haven't enabled the new design language?

Reply:
@dleroux was this ever working correctly though? There was a comment in the code about doing _something a certain way_ in order to animate the `background: linear-gradient`, but as far as I know, you can't animate gradients. So we had some properties animating while others were not, adding up to a disjointed experience.

Worth mentioning though - the properties that were animating correctly were things like `border-color` and `box-shadow`, and I think they were hard to notice in the first place. So the experience wasn't terribly inconsistent, unless you were _really paying attention_.

Essentially, I don't really see the drawback of doing this for old design language users. But I completely defer to Polaris here - you folks are going to have way better context than I will 😆

Reply:
Ha yes, It was a safari bug IIRC. @danrosenthal and @tmlayton were involved in that.

> you folks are going to have way better context than I will.

I do have context but I'm not on Polaris anymore :)

Reply:
No issues related to the Safari bug here. That was related to the `linear-gradient` function being passed as a `var` function fallback property.